modules/telemetry/v1/data_block.py

```diff
@@ -307,6 +309,93 @@ def __iter__(self):
         yield "mission_time", self.mission_time
         yield "percentage", round(self.humidity / 100)
 
+class LinearAccelerationDB(DataBlock):
+    """Represents a linear acceleration data block"""
+
+    def __init__(self, mission_time: int, x_axis: int, y_axis: int, z_axis: int) -> None:
+        """
+        Constructus a linear acceleration data block.
+        
+        Args:
+            mission_time: The mission time the linear acceleration was measured in milliseconds since launch.
+            x_axis: The acceleration about the x axis in meters per second squared.
+            y_axis: The acceleration about the y axis in meters per second squared.
+            z_axis: The acceleration about the z axis in meters per second squared.
+            
+        """
+        super().__init__(mission_time)
+        self.x_axis: int = x_axis
+        self.y_axis: int = y_axis
+        self.z_axis: int = z_axis
+
+    @classmethod
+    def from_bytes(cls, payload: bytes) -> Self:
+        """
+        Constructs a linear acceleration data block from bytes.
+        Returns:
+            A linear acceleration data block.
+        """
+        parts = struct.unpack("<Ihhh", payload)
```

Comment:
If testing fails, it may be because you also need to unpack the final 16bit member which is just 0 padding. Not sure, this might work as-is.